modules/rosa-sdpolicy-am-regions-az.adoc

```diff
@@ -37,9 +37,10 @@ ifndef::rosa-with-hcp[]
 * ap-south-1 (Mumbai)
 * ap-south-2 (Hyderabad, AWS opt-in required)
 * ap-southeast-1 (Singapore)
-* ap-southeast-2 (Sydney)
 endif::rosa-with-hcp[]
+* ap-southeast-2 (Sydney)
 * ap-southeast-3 (Jakarta, AWS opt-in required)
+* ap-southeast-4 (Melbourne, AWS opt-in required)
 ifndef::rosa-with-hcp[]
```

Comment:
There's a duplicated zone here on the ROSA Classic side

![image](https://github.com/openshift/openshift-docs/assets/16167833/cc58a07f-2e81-4360-ba74-159e0aae85cc)

That isn't present on ROSA with HCP:

![image](https://github.com/openshift/openshift-docs/assets/16167833/248494bb-2978-47d0-8f4e-e3ac0b121b02)